src/core/components/InstallButton/styles.scss

```diff
@@ -22,3 +22,7 @@
     }
   }
 }
+
+.InstallButton-button .Icon-plus {
+  vertical-align: baseline;
```

Comment:
Surprisingly, I find this button prettier **without** this property.

With:

<img width="170" alt="screen shot 2017-10-31 at 11 04 26" src="https://user-images.githubusercontent.com/217628/32218609-c0ee3f48-be2b-11e7-9585-e8ef6d98e962.png">

Without: (the icon seems better centered)

<img width="172" alt="screen shot 2017-10-31 at 11 04 19" src="https://user-images.githubusercontent.com/217628/32218608-c0d3b3b2-be2b-11e7-8113-37b6c555a524.png">

Reply:
According to the mockups, the button should be more rounded (sketch says 12px) and there should be a box shadow.

Reply:
Yeah, there should be a fair amount of tweaks to the buttons but I wanted to do that in a separate PR as it'll touch a fair bit of code/pages and there are a lot of tweaks to be done.

Reply:
How does it look for the "Install theme" button though? I agree, but I think it made the themes button look really off. I guess I could make it only appear for themes.

Reply:
I am fine with doing this later, things are quite consistent currently so let's avoid visually breaking them.